### PR TITLE
Rename to `MaxEcdsaAuthorities`

### LINKED ISSUE #1388: Fix broken test script
```sh
./tests/local-testnet/deploy.sh pangolin

2022-08-10 13:20:20 It isn't safe to expose RPC publicly without a proxy server that filters available set of RPC methods.    
2022-08-10 13:20:20 It isn't safe to expose RPC publicly without a proxy server that filters available set of RPC methods.    
2022-08-10 13:20:20 Darwinia Runtime Module Library    
2022-08-10 13:20:20 ✌️  version 2.8.19-5d454d55-x86_64-linux-gnu    
2022-08-10 13:20:20 ❤️  by Darwinia Network <hello@darwinia.network>, 2018-2022    
2022-08-10 13:20:20 📋 Chain specification: Pangolin Local Testnet    
2022-08-10 13:20:20 🏷 Node name: Alice    
2022-08-10 13:20:20 👤 Role: AUTHORITY    
2022-08-10 13:20:20 💾 Database: RocksDb at /home/bear/working/darwinia/darwinia-common/tests/local-testnet/data/alice/chains/pangolin_dev/db/full    
2022-08-10 13:20:20 ⛓  Native runtime: Pangolin-28190 (Pangolin-0.tx0.au0)    
2022-08-10 13:20:20 `pallet_timestamp::UnixTime::now` is called at genesis, invalid value returned: 0    
2022-08-10 13:20:20 `pallet_timestamp::UnixTime::now` is called at genesis, invalid value returned: 0    
2022-08-10 13:20:20 `pallet_timestamp::UnixTime::now` is called at genesis, invalid value returned: 0    
2022-08-10 13:20:20 `pallet_timestamp::UnixTime::now` is called at genesis, invalid value returned: 0    

====================

Version: 2.8.19-5d454d55-x86_64-linux-gnu

   0: sp_panic_handler::set::{{closure}}
   1: std::panicking::rust_panic_with_hook
             at /rustc/6dd68402c5d7da168f87d8551dd9aed1d8a21893/library/std/src/panicking.rs:702:17
   2: std::panicking::begin_panic_handler::{{closure}}
             at /rustc/6dd68402c5d7da168f87d8551dd9aed1d8a21893/library/std/src/panicking.rs:588:13
   3: std::sys_common::backtrace::__rust_end_short_backtrace
             at /rustc/6dd68402c5d7da168f87d8551dd9aed1d8a21893/library/std/src/sys_common/backtrace.rs:138:18
   4: rust_begin_unwind
             at /rustc/6dd68402c5d7da168f87d8551dd9aed1d8a21893/library/std/src/panicking.rs:584:5
   5: core::panicking::panic_fmt
             at /rustc/6dd68402c5d7da168f87d8551dd9aed1d8a21893/library/core/src/panicking.rs:142:14
   6: core::result::unwrap_failed
             at /rustc/6dd68402c5d7da168f87d8551dd9aed1d8a21893/library/core/src/result.rs:1785:5
   7: pallet_babe::<impl pallet_babe::pallet::Pallet<T>>::initialize_authorities
   8: <(TupleElement0,TupleElement1,TupleElement2,TupleElement3,TupleElement4) as pallet_session::SessionHandler<AId>>::on_genesis_session
   9: <pallet_session::pallet::GenesisConfig<T> as frame_support::traits::hooks::GenesisBuild<T>>::build
  10: std::thread::local::LocalKey<T>::with
  11: sp_state_machine::basic::BasicExternalities::execute_with_storage
  12: <pangolin_runtime::GenesisConfig as sp_runtime::BuildStorage>::assimilate_storage
  13: sp_runtime::BuildStorage::build_storage
  14: <sc_chain_spec::chain_spec::ChainSpec<G,E> as sp_runtime::BuildStorage>::build_storage
  15: sc_service::client::client::Client<B,E,Block,RA>::new
  16: sc_service::builder::new_full_parts
  17: drml_node_service::service::new_full
  18: drml_node_service::service::pangolin::new_full
  19: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll
  20: std::thread::local::LocalKey<T>::with
  21: tokio::park::thread::CachedParkThread::block_on
  22: tokio::runtime::thread_pool::ThreadPool::block_on
  23: tokio::runtime::Runtime::block_on
  24: drml::command::run
  25: drml::main
  26: std::sys_common::backtrace::__rust_begin_short_backtrace
  27: std::rt::lang_start::{{closure}}
  28: core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &F>::call_once
             at /rustc/6dd68402c5d7da168f87d8551dd9aed1d8a21893/library/core/src/ops/function.rs:280:13
      std::panicking::try::do_call
             at /rustc/6dd68402c5d7da168f87d8551dd9aed1d8a21893/library/std/src/panicking.rs:492:40
      std::panicking::try
             at /rustc/6dd68402c5d7da168f87d8551dd9aed1d8a21893/library/std/src/panicking.rs:456:19
      std::panic::catch_unwind
             at /rustc/6dd68402c5d7da168f87d8551dd9aed1d8a21893/library/std/src/panic.rs:137:14
      std::rt::lang_start_internal::{{closure}}
             at /rustc/6dd68402c5d7da168f87d8551dd9aed1d8a21893/library/std/src/rt.rs:128:48
      std::panicking::try::do_call
             at /rustc/6dd68402c5d7da168f87d8551dd9aed1d8a21893/library/std/src/panicking.rs:492:40
      std::panicking::try
             at /rustc/6dd68402c5d7da168f87d8551dd9aed1d8a21893/library/std/src/panicking.rs:456:19
      std::panic::catch_unwind
             at /rustc/6dd68402c5d7da168f87d8551dd9aed1d8a21893/library/std/src/panic.rs:137:14
      std::rt::lang_start_internal
             at /rustc/6dd68402c5d7da168f87d8551dd9aed1d8a21893/library/std/src/rt.rs:128:20
  29: main
  30: <unknown>
  31: __libc_start_main
  32: _start
             at /build/glibc/src/glibc/csu/../sysdeps/x86_64/start.S:115


Thread 'main' panicked at 'Initial number of authorities should be lower than T::MaxAuthorities: ()', /home/bear/.cargo/git/checkouts/substrate-a7fa553ead635512/69ac763/frame/babe/src/lib.rs:777

This is a bug. Please report it at:

	https://github.com/darwinia-network/darwinia-common/issues/new

2022-08-10 13:20:20 [0] 💸 generated 4 npos voters, 4 from validators and 0 nominators    
2022-08-10 13:20:20 [0] 💸 generated 4 npos voters, 4 from validators and 0 nominators    
2022-08-10 13:20:20 [0] 💸 new validator set of size 4 has been processed for era 1    
```

```sh
./tests/local-testnet/deploy.sh pangoro

Error: Input("Error opening spec file `pangoro-local`: No such file or directory (os error 2)")
```

### PR DESCRIPTION
**Pull request by AurevoirXavier**
Close #1388.

### CODE REVIEW

File: node/runtime/pangolin/src/pallets/ecdsa_authority.rs [hunk 1]
```diff
@@ -17,7 +17,7 @@ static_assertions::const_assert!(SyncInterval::get() < MaxPendingPeriod::get());
 impl Config for Runtime {
 	type ChainId = ChainId;
 	type Event = Event;
-	type MaxAuthorities = MaxAuthorities;
+	type MaxAuthorities = MaxEcdsaAuthorities;
```
Comment (AsceticBear) [inline node/runtime/pangolin/src/pallets/ecdsa_authority.rs:20]: Suggest renaming `type MaxAuthorities` to `type MaxEcdsaAuthorities`.

Comment (AurevoirXavier) [reply node/runtime/pangolin/src/pallets/ecdsa_authority.rs:20]: I think `MaxAuthorities` is better.

Comment (AsceticBear) [reply node/runtime/pangolin/src/pallets/ecdsa_authority.rs:20]: I see. It is easy to make mistakes with the property in the grandpa pallet, which can lead to configuration errors, especially for novices.  I just have this concern.

Comment (AurevoirXavier) [reply node/runtime/pangolin/src/pallets/ecdsa_authority.rs:20]: I don't think so.
It is under the `darwinia_ecdsa_authority::Config`.
It is not easy to make mistakes here.

Just like `type PalletId = XXX`.
No need to specify
```
dvm::Config	{ type DvmPalletId }
staking::Config {type StakingPalletId }
treasury::Config { type TreasuryPalletId }
```

**Review by AsceticBear — APPROVED** [review]

**Pull request merged**